src/technologies/n.json

```diff
@@ -682,6 +682,39 @@
     "requires": "Next.js",
     "website": "https://nextjs.org/docs/app"
   },
+  "Next.js SSR": {
+    "cats": [
+      12
```

Comment:
```suggestion
      22
```

Reply:
I think we should also remove 22 from Next.js above

Reply:
I've done that, but based on https://github.com/HTTPArchive/wappalyzer/pull/6#issuecomment-1854628880 I've included them in App Router as they might be SSG